CONTRIBUTING.md

```diff
@@ -21,30 +21,35 @@ We encourage you to submit pull requests (PRs) to improve our project. To do so,
    ```bash
    git clone https://github.com/Arch4ngel21/scikit-fingerprints.git
 
-3. If you don't have it, install poetry:
+3. If you don't have it, install poetry (preferably with pipx):
    ```bash
-   pip install poetry
+   pipx install poetry
```

Comment:
I wouldn't explicitly mention this, since this is not directly relevant to the library. Also, what if Poetry changes something here? I think avoiding such explicit dependencies would be a good idea

Reply:
I thought of mentioning it, because it is explicitly advised against using bare pip to install poetry in official poetry docs [(reference (red ban sign))](https://python-poetry.org/docs/#installation).
Mentioning pipx may not be the best idea, but I still think that it is better than suggesting doing explicitly discouraged actions. Alternatively, we could just mention that poetry is required, with no suggestion how to install it

Reply:
Yes, I think we should not mention anything besides that Poetry is required. We shouldn't care how developer installs it.